frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -272,7 +272,7 @@ class PublicDashboardInner extends Component<PublicDashboardProps> {
                   isPublic
                   mode={PublicMode as unknown as Mode}
                   metadata={this.props.metadata}
-                  navigateToNewCardFromDashboard={() => {}}
+                  navigateToNewCardFromDashboard={undefined}
```

Comment:
Can we just skip the prop altogether? 🤔

Reply:
7e9777b5f6e3adb834e3861763226c7c1e918675